### PR TITLE
Add the scripts for the modal

### PR DESCRIPTION
**Pull request by Bestbynature**
⚡ Features include:

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,3 +3,43 @@ const modal = document.querySelector('.modal');
...
+  if ((event.target === modal) || (event.target === links)) {
+    modalHide();
+  }
+}
+
+function modalShow() {
+  modal.style.display = 'flex';
+  wrapper.style.filter = 'blur(0.4rem)';
+  modal.scrollIntoView({ behavior: 'smooth' });
+}
```
Comment (omarsalem7) [inline index.js:22]: - [ ] Kindly consider making the modal take the height of the screen by changing the height of the modal to `100vh` at [this line](https://github.com/Bestbynature/portfolio/blob/15a22bc86d493c35a0523641962692d8678559bd/styles.css#L87)
- [ ] Also consider sticking to the design by using the correct font size and also make sure to put the close icon in the proper position like the Figma design to make your design identical to Figma

| Your design  | Figma design|
| ------------- | ------------- |
| image | image |

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Bestbynature 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Pop modal open and close properly ✔️
- The Pull Request has a proper title and description. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @Bestbynature   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] As requested by the previous reviewer, kindly make sure your mobile menu covers the whole screen height and make sure even if we scroll down the mobile menu will still cover the whole weight of the visible screen.

   - ### APP BEHAVIOUR 
   chrome-capture-2023-2-6
  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bestbynature** [comment]
@levy002 , Thank you for the point you raised. But as he advised, I changed the height from 95vh to 100vh. Are you suggesting i increase it from 100 to something more??

**Review by nicupop729 — COMMENTED** [review]
Hello @Bestbynature and @melybee-07  👋,

_This is an additional review as a result of the second opinion request._

Kudos for the good project you built here!

Because what was asked by the first reviewer was implemented, correctly would have been that the second reviewer to approve the CRR. 
He still asked it because the feature for was not still working. More refactoring is required than to just add 100vh to the container. 
@omarsalem7 should have research a bit more the code in order to give an accurate advice.   

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @Bestbynature 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


# Highlights
- No linter errors ✔️
- Good implementations ✔️


## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.


__check the comments for details

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**